pkg/apis/eksctl.io/v1alpha5/assets/schema.json

```diff
@@ -577,6 +577,12 @@
             "IPv6"
           ]
         },
+        "ipv6Cidr": {
+          "type": "string"
+        },
+        "ipv6CidrPool": {
```

Comment:
Looks good :) I realise how late it is to make this change, but could we rename the new `ipv6CidrPool` field to `ipv6Pool`? We should have caught this way earlier (we also probably didn't know as much as we do now) but I don't think there's such a thing as a `CIDR pool`: it's really just an `address pool` and the CIDR range gets derived from it.

Reply:
yeah sure!

Reply:
yep, `ipv6Pool ` checks out: https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_Ipv6Pool.html not sure how this slipped through!

Reply:
done!